Flutter/calendar/schedule-view.md

```diff
@@ -22,7 +22,7 @@ The schedule view display two different UI for mobile and web, for mobile it wil
 You can customize the height of an appointment in a schedule view by using the [appointmentItemHeight](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/ScheduleViewSettings/appointmentItemHeight.html) property of [ScheduleViewSettings](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/ScheduleViewSettings-class.html).
 
 {% tabs %}
-{% highlight dart hl_lines="7" %}
+{% highlight dart hl_lines="6 7 8" %}
```

Comment:
Highlight scheduleviewsettings not needed

Reply:
Unwanted highlights removed.